packages/gateway/src/constants.ts

```diff
@@ -16,7 +16,7 @@ export const AVATAR_API =
 
 export const NATIVE_LANGUAGE = (() => {
   const languages = env.LANGUAGES.split(',')
-  return languages.find((language) => language !== 'en')
+  return languages.find((language: string) => language !== 'en')
```

Comment:
![image](https://github.com/user-attachments/assets/d6361d7f-73a3-458c-a27f-19bf705046d0)
This is an unnecessary assertion: the compiler seems to already know it's a string.

Reply:
the parameter "language" was being implicitly given an any type which was causing an error when i ran the tests